frontend/components/commons/forms/questions-form/fields/text-area/TextArea.component.vue

```diff
@@ -0,0 +1,113 @@
+<template>
+  <div class="wrapper">
+    <div class="title-area --body1">
+      <span
+        :key="colorHighlight"
+        v-text="title"
+        v-required-field="isRequired ? { color: colorHighlight } : false"
+      />
+
+      <TooltipComponent
+        v-if="isIcon"
+        :message="tooltipMessage"
+        direction="bottom"
+      >
+        <svgicon class="icon" name="info" width="22" height="22" />
+      </TooltipComponent>
+    </div>
+
+    <div class="container">
+      <Text2TextContentEditable
```

Comment:
For now, I reuse things to match the goal to deliver something for the next week first iteration. But I agree that it will be nice to do a specific component for that in a refactoring after this iteration